Source/Core/VideoBackends/Vulkan/VKDebug.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <memory>
+#include <optional>
+#include <string_view>
+
+#include "Common/CommonTypes.h"
+#include "VideoBackends/Vulkan/Constants.h"
+#include "VideoBackends/Vulkan/StagingBuffer.h"
+#include "VideoCommon/AbstractGfx.h"
+#include "VideoCommon/Constants.h"
+
+#define EMIT_VK_DEBUG_MARKER(CMD, AUX0, AUX1, AUX2, AUX3)                                          \
```

Comment:
Any reason this can't be an inline (or prototype/imp pair) function in a header within a namespace?

Reply:
No reason, that would be cleaner, so I'm gonna change it.